conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -133,7 +133,10 @@ public HttpHandler build() {
         private void checkOverlappingPaths() {
             List<String> duplicates = endpoints.stream()
                     .collect(Collectors.groupingBy(
-                            endpoint -> String.format("%s: %s", endpoint.method(), endpoint.template())))
+                            endpoint -> String.format(
+                                    "%s: %s",
+                                    endpoint.method(),
+                                    endpoint.template().replaceAll("\\{.*\\}", "{}"))))
```

Comment:
I think we want `.*?` so that `/{a}/{b}` is normalized to `/{}/{}`, not `/{}`. We should add a test. I can push this update, you should enjoy your weekend!
Might use `{param}` as the replacement for cleaner error messages.

Reply:
Nice catch. param sounds good, will do that and :)